gpu-lab.md

```diff
@@ -7,22 +7,51 @@ title: Welcome to the CHTC GPU Lab
 The CHTC GPU Lab is a [UW2020-funded project][uw2020] that will expand shared GPU 
 computing infrastructure at UW-Madison.  It will include:
 
-* a pool of shared GPU servers managed by CHTC 
-* a community of technical experts
-* a library of sharable software and documentation
+ <link rel = "stylesheet"
+   type = "text/css"
+   href = "bootstrap.css" />
 
-To get started with GPU computing in CHTC, visit the [GPU jobs guide][gpu-jobs] 
-and see the [runnable examples on GitHub][gpu-examples].
+<div class="card-deck">
+	<div class="card border-secondary h-100" >    
+		 <h5 class="card-title text-center">Hardware</h5>
+		 <img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/noun_gpu_2528527.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
+		<p style="padding:10px">A pool of shared GPU servers managed by CHTC.</p>
+	</div>
+	<div class="card border-secondary h-100" >    
+		 <h5 class="card-title text-center">Expertise</h5>
+		 <img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/noun_people_1188645.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
+		<p style="padding:10px">A community of technical experts.</p>
+	</div>
+	<div class="card border-secondary h-100" >    
+		 <h5 class="card-title text-center">Documentation</h5>
+		 <img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/noun_open book_1179297.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
+		<p style="padding:10px">A library of sharable software and documentation.</p>
+	</div>
+</div>
 
-For more information about the CHTC GPU Lab contact [Anthony Gitter][gitter]. Email 
-[chtc@cs.wisc.edu](mailto:chtc@cs.wisc.edu) for general CHTC GPU computing 
-questions.  Subscribe to the CHTC GPU Lab WiscList 
-by emailing [join-chtc-gpu-lab@lists.wisc.edu](mailto:join-chtc-gpu-lab@lists.wisc.edu) to receive updates.
+## Get Involved
 
-UW2020 is funded by the Office of the Vice Chancellor for Research and Graduate 
-Education and the Wisconsin Alumni Research Foundation. 
+If you want to use GPU resources in CHTC for your research: 
 
+- Apply for a CHTC account if you don't already have one: [Account Request Page][account]
```

Comment:
Is this the right link?  The link format looks funny too...

Maybe we should also ask them to indicate they are interested in GPUs in that form?

Reply:
Ah, nevermind.  We're using the reference-style format.  Just not used to seeing that.